samples/bluetooth/peripheral_uart/prj.conf

```diff
@@ -39,3 +44,12 @@ CONFIG_DK_LIBRARY=y
 
 # This example requires more workqueue stack
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Config logger
+CONFIG_LOG=y
+CONFIG_LOG_MINIMAL=n
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_PRINTK=n
+CONFIG_LOG_DEFAULT_LEVEL=3
```

Comment:
This is also the default

Reply:
Default value of this is 1(ERROR). In default configuration only error logs are printed